src/GogeDao.sol

```diff
@@ -370,718 +476,955 @@ contract GogeDAO is Ownable {
+        }
+        else if (pollTypes[_pollNum] == PollType.setTeamEnabled) {
+            SetTeamEnabled memory setTeamEnabled;
+            (,setTeamEnabled,) = getSetTeamEnabled(_pollNum);
+            IGogeERC20(governanceTokenAddr).setTeamEnabled(setTeamEnabled.boolVar);
+        }
+        else if (pollTypes[_pollNum] == PollType.updateUniswapV2Router) {
+            UpdateUniswapV2Router memory updateUniswapV2Router;
+            (,updateUniswapV2Router,) = getUpdateUniswapV2Router(_pollNum);
+            IGogeERC20(governanceTokenAddr).updateUniswapV2Router(updateUniswapV2Router.addr);
+        }
+        else if (pollTypes[_pollNum] == PollType.excludeFromFees) {
+            ExcludeFromFees memory excludeFromFees;
+            (,excludeFromFees,) = getExcludeFromFees(_pollNum);
+            IGogeERC20(governanceTokenAddr).excludeFromFees(excludeFromFees.addr, excludeFromFees.boolVar);
+        }
+        else if (pollTypes[_pollNum] == PollType.excludeFromDividends) {
+            ExcludeFromDividends memory excludeFromDividends;
+            (,excludeFromDividends,) = getExcludeFromDividends(_pollNum);
+            IGogeERC20(governanceTokenAddr).excludeFromDividend(excludeFromDividends.addr);
+        }
+        else if (pollTypes[_pollNum] == PollType.modifyBlacklist) {
+            ModifyBlacklist memory modifyBlacklist;
+            (,modifyBlacklist,) = getModifyBlacklist(_pollNum);
+            IGogeERC20(governanceTokenAddr).modifyBlacklist(modifyBlacklist.addr, modifyBlacklist.blacklisted);
+        }
+        else if (pollTypes[_pollNum] == PollType.transferOwnership) {
+            TransferOwnership memory transferOwnership;
+            (,transferOwnership,) = getTransferOwnership(_pollNum);
+            IGogeERC20(governanceTokenAddr)._transferOwnership(transferOwnership.addr);
+        }
+        else if (pollTypes[_pollNum] == PollType.migrateTreasury) {
+            MigrateTreasury memory migrateTreasury;
+            (,migrateTreasury,) = getMigrateTreasury(_pollNum);
+            IGogeERC20(migrateTreasury.token).transfer(migrateTreasury.addr, IGogeERC20(migrateTreasury.token).balanceOf(address(this)));
+        }
+        else if (pollTypes[_pollNum] == PollType.setQuorum) {
+            SetQuorum memory setQuorum;
+            (,setQuorum,) = getSetQuorum(_pollNum);
+            _updateQuorum(setQuorum.amount);
+        }
+        else if (pollTypes[_pollNum] == PollType.setMinPollPeriod) {
+            SetMinPollPeriod memory setMinPollPeriod;
+            (,setMinPollPeriod,) = getSetMinPollPeriod(_pollNum);
+            _updateMinPollPeriod(setMinPollPeriod.amount);
+        }
+        else if (pollTypes[_pollNum] == PollType.updateGovernanceToken) {
+            UpdateGovernanceToken memory updateGovernanceToken;
+            (,updateGovernanceToken,) = getUpdateGovernanceToken(_pollNum);
+
+            _removePollFromActivePolls(_pollNum);
```

Comment:
Reason being when we call _refundVoters() it will send all tokens of governanceTokenAddr to the voters. However, when we call _changeGovernanceToken we are updating that value to a new address of which this contract balance is 0. So, when we go to refund the voters it will revert since it's now trying to refund voters a token that they (1. did not vote with, 2. This contract has a 0 balance). So, we refund voters THEN change the governance token, then 'return' ending the function call